components/calendar/src/helpers.rs

```diff
@@ -59,6 +59,16 @@ pub const fn quotient64(n: i64, d: i64) -> i64 {
     }
 }
 
+/// Return y if x.rem_euclid(y) == 0, x.rem_euclid(y) otherwise
+pub fn adjusted_rem_euclid(x: i32, y: i32) -> i32 {
```

Comment:
Issue 1: are you using this function anywhere? I don't see it in the diff.

Issue 2: we should move away from i32 math and favor i64 math in most cases moving forward.

Reply:
This was not supposed to be here, so I have removed it